getting_started/workflow/best_practices/scenes_versus_scripts.rst

```diff
@@ -0,0 +1,181 @@
+.. _doc_scenes_versus_scripts:
+
+When to use scenes versus scripts
+=================================
+
+We've already covered how scenes and scripts are different. Scripts
+define an engine class extension with imperative code, scenes with
+declarative code.
+
+Each system's capabilities are different as a result.
+Scenes can define how an extended class initializes, but not what its
+behavior actually is. Scenes are often used in conjunction with a script so
+that the scene acts as an extension of the scripts declarative code.
+
+Anonymous types
+---------------
+
+It *is* possible to completely define a scenes' contents using a script alone.
+This is, in essence, what the Godot Editor does, only in the C++ constructor
+of its objects. 
+
+But, choosing which one to use can be a dilemma. Creating script instances
+is identical to creating in-engine classes whereas handling scenes requires
+a change in API:
+
+    .. tabs::
+      .. code-tab:: gdscript GDScript
+
+        const MyNode = preload("my_node.gd")
+        const MyScene = preload("my_scene.tscn")
+        var node = Node.new()
+        var my_node = MyNode.new() # Same method call
+        var my_scene = MyScene.instance() # Different method call
+        var my_inherited_scene = MyScene.instance(PackedScene.GEN_EDIT_STATE_MAIN) # Create scene inheriting from MyScene
+
+      .. code-tab:: csharp
+
+        using System;
+        using Godot;
+
+        public class Game : Node
+        {
+            public const Script MyNodeScr = ResourceLoader.load("MyNode.cs") as Script;
+            public const PackedScene MySceneScn= ResourceLoader.load("MyScene.tscn") as PackedScene;
+            public Node ANode;
+            public Node MyNode;
+            public Node MyScene;
+            public Node MyInheritedScene;
+
+            public Game()
+            {
+                ANode = new Node();
+                MyNode = new MyNode(); // Same syntax
```

Comment:
Now that I think about it, would this even work? Is the MyNode name exposed to other C# scripts just because they are in the same project? And does `MyNodeScr.new()` do the same thing that `new MyNode()` would do? I'm not good with Godot C#.